schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
```

Comment:
- [OPTIONAL]
 you can set the ID to auto increment by `id int NOT NULL IDENTITY(1,1)`